npm/webpack-preprocessor/index.ts

```diff
@@ -352,7 +350,9 @@ const preprocessor: WebpackPreprocessor = (options: PreprocessorOptions = {}): F
 
       if (file.shouldWatch) {
         // in this case the bundler is webpack.Compiler.Watching
-        (bundler as webpack.Compiler.Watching).close(cb)
+        if (bundler && 'close' in bundler) {
+          bundler.close(cb)
```

Comment:
Is there cleanup that we need to do if `close` doesn't exist?

Reply:
Not as far as I know. Have you ran into any problems around webpack cleanup before?

Reply:
I haven't, but I also haven't modified/managed Webpack itself before